kratos/python/add_matrix_to_python.cpp

```diff
@@ -89,6 +90,25 @@ namespace Python
 
           return matrix;
         }));
+
+        matrix_binder.def(py::init([](const py::list& input){
+          std::size_t num_rows = input.size();
+          KRATOS_ERROR_IF( num_rows == 0 ) << "Argument must be a Python's list of lists\n";
```

Comment:
So, if input is like `C = KM.Matrix([]) `
Will get `RuntimeError: Error: Argument must be a Python's list of lists`